tools/getkeys/package.json

```
@@ -14,11 +14,20 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": "eslint ./index.js",
+    "eslint-fix": "eslint ./index.js --fix",
+    "lint": "npm run eslint",
+    "lint-fix": "npm run eslint-fix"
```

Let's keep the standard script names across the repo, with colons instead of dashes. I think our tooling that runs during CI will actually validate that these are the exact names of the scripts.
```suggestion
    "eslint": "eslint ./index.js",
    "eslint:fix": "eslint ./index.js --fix",
    "lint": "npm run eslint",
    "lint:fix": "npm run eslint:fix"
```